component/src/test/resources/testng.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~      http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied. See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Siddhi-Io-Cdc-Test-Suite">
+    <test name="Siddhi-io-cdc-tests" enabled="false" preserve-order="true" parallel="false">
+        <classes>
+            <class name="org.wso2.extension.siddhi.io.cdc.source.TestCaseOfCDCSource"/>
```

Comment:
Skip test cases if it depends on any depended databases that need to be run in the build machine

Reply:
Fixed with commit: `Comment-out test: "Siddhi-io-cdc-tests".` d834e69